### DISCUSSION TITLE
How far behind Angular releases do we expect nx to be?

### DISCUSSION THREAD

**Discussion by markgoho** [nrwl/nx#4773] [General]
I'm on the latest version of nx/workspace, but my @angular/core version is _two minor versions_ behind. Is this acceptable to the community? Are we okay with this kind of delay moving forward?

**Comment by beeman** [comment]
Just for your info, this just landed https://github.com/nrwl/nx/pull/4788

**Reply by markgoho** [reply]
Thanks @beeman! 

@brandonroberts Are 17 file changes going to be required every time we want to update to a new release of Angular?

**Reply by brandonroberts** [reply]
@markgoho no, I doubt that will be the case on every update. To answer your other question about the latest version, _it depends_. We try to get updates in as quickly as possible, along with bug fixes and other work. Each update requires additional testing to try to ensure we don't break any workspaces.

**Reply by markgoho** [reply]
@brandonroberts is there any way to make the process a bit clearer so that any community member could potentially update Angular to a new version? Is that fantasy-land?
